crmsh/corosync.py

```diff
@@ -811,20 +811,48 @@ class IPAlreadyConfiguredError(Exception):
     pass
 
 
-def add_node_ucast(IParray, node_id=None):
-
-    f = open(conf()).read()
-    p = Parser(f)
-
-    # to check if the same IP already configured
+def find_configured_ip(IParray):
+    """
+    find if the same IP already configured
+    If so, raise IPAlreadyConfiguredError
+    """
+    with open(conf()) as f:
+        p = Parser(f.read())
+
+    # get exist ip list from corosync.conf
     exist_iplist = []
```

Comment:
I would change this variable names. `exist_iplist` by `corosync_ip_list`

Reply:
changed